game_eggs/steamcmd_servers/valheim/valheim_bepinex/egg-valheim-bep-i-nex.json

```diff
@@ -0,0 +1,184 @@
+            "container": "ghcr.io\/pteroforge\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Server Name",
+            "description": "Name that appears in server browser.",
+            "env_variable": "SERVER_NAME",
+            "default_value": "My Server",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:60",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Password",
+            "description": "Server password.",
+            "env_variable": "PASSWORD",
+            "default_value": "secret",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|min:5|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "World Name",
+            "description": "Name to load if switching between multiple saved worlds.",
+            "env_variable": "WORLD",
+            "default_value": "Dedicated",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Public Server",
+            "description": "",
+            "env_variable": "PUBLIC",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20",
```

Comment:
if this is 1 and 0 then this ca, be `required|string|in:0,1`  or `required|boolean`then it will be transferd to the sitch

Reply:
thats from the other eggs :D

Reply:
oké then that is fine